docs/modules/ROOT/pages/how-tos/workflows/keep-remote-pipelines-up-to-date.adoc

```diff
@@ -0,0 +1,107 @@
+= Maintaining remote Tekton pipelines up to date
+
+[IMPORTANT]
+====
+* This article took for granted the reader knows what a link:https://tekton.dev/docs/pipelines/resolution/[Tekton remote pipeline] is. If this is not the case, please read more about this feature before proceeding.
+====
+
+== The advantage of using a remote pipeline
+
+This approach allows a team or an individual contributor to manage plenty of Pipelines from a single repository as source of truth. But there also other good reasons:
+
+. Pipelines are stored in a Git repository and fetched during execution, eliminating the need to maintain local or static pipeline definitions.
+. Teams can leverage Git’s versioning to access specific pipeline branches
+. In case of issues, it's very simple to rollback to a previous pipeline version, and apply it globally among the Konflux Components.
+. Reduced maintenance overhead
```

Comment:
```suggestion
. Reduced maintenance overhead of keeping task references up to date across multiple components.
```

Reply:
There is more maintenance overhead. For example: if you need custom tasks in all of your pipelines, if you manage different versions through the project controller you need to update all branches as well if you change something in a task

Reply:
Mmmh, why would you target a different revision/branch everywhere? Can't you just test your changes on a branch updating a single component and then open a PR against main/devel branch?